app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -66,9 +120,13 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  @Ignore("Reenable once OngoingStoryList in domain layer is updated.") // TODO(#735): Reenable once OngoingStoryList is fetched correctly.
   fun testRecentlyPlayedTestActivity_recyclerViewItem0_doesNotShowSectionDivider() {
-    ActivityScenario.launch(RecentlyPlayedFragmentTestActivity::class.java).use {
+    ActivityScenario.launch<RecentlyPlayedActivity>(createRecentlyPlayedActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_story_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          1
```

Comment:
scrollToPosition - 1 does not match with the method name which says `recyclerViewItem0`. Please either correct the method's name or write the functionality as the method name suggested.

Reply:
Resolved.